README.md

```diff
@@ -1 +1,43 @@
-# graasp-template-backend
+# graasp-plugin-validation
+
+# Purpose
+The Graasp Explorer is designed to be a platform to share and explore open educational resources. Inappropriate content, such as dirty words, aggressive langauge, hate speech and etc., should be prohibited from publishing.
+
+# Introduction
+## Validation Process
+There will be different validation processes to check different things. Each validation process will have a record in DB.
+
+### Existing
+- Bad Words Detection (JS)
+### In progress
+- Aggressive and Hate Speech Classification (Python)
+- Image Checking
+
+## Status
+
+### Status for item-validation
+
+**Pending**: The process is in progress.
+
+**Success**: The item passes the test and can be published.
+
+**Failure**: The item is suspicious, and could not be published at the moment. A failed process will wait for manual review from administrator.
+
+### Status for item-validation-review
+
+**Pending**: The item-validation record is waiting for manual review.
+
+**Accepted**: The admin accepts the result of the validation, which means that the item is indeed problematic and the publication cannot proceed
+
+**Rejected**: Admin reject the result of validation => The item is OK, and will be published.
+The admin rejects the result of the validation, which means that the item is OK and the publication cannot proceed
+
+# Endpoints
+## Start Validation
+`POST /validations/:itemId` - A new validation job will start on given item. Enabled validation processes will start to run on given item. After each process finishes, it will update corresponding record in `item-validation` with results. If an automatic process fails, it will then create a new entry in `item-validation-review`.
+
+`GET /validations/reviews` - Returns all entries in `item-validation-review` that has a `pending` status. This endpoint is mainly used in admin mode to display all items that need manual review.
```

Comment:
Actually I would return all reviews, and the frontend decides how to manage the pending ones, what do you think?

Reply:
Yea I think we can do that for now. But maybe when we get more and more items, some day in the future we might want to only get the pending ones, otherwise it could take very long.